template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/.template.config/template.json

```diff
@@ -267,11 +267,61 @@
       "type": "computed",
       "value": "(OrganizationalAuth || IndividualAuth)"
     },
-    "UseLocalDB": {
+    "EFProvider": {
       "type": "parameter",
-      "datatype": "bool",
-      "defaultValue": "false",
-      "description": "Whether or not to use LocalDB instead of SQLite"
+      "datatype": "choice",
+      "choices": [
+        {
+          "choice": "InMemory",
+          "description": "This database provider allows Entity Framework Core to be used with an in-memory database. This is useful when testing code that uses Entity Framework Core. The provider is maintained as part of the EntityFramework GitHub project."
```

Comment:
I'd assert stronger that this provider is only intended for testing (and even then, we recommend using SQLite in-memory databases). Maybe it shouldn't even be an option...